tcl/target/snps_hs5_fpga.cfg

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+#  Copyright (C) 2022 Synopsys, Inc.
+#  Shahab Vahedi <shahab@synopsys.com>
+
+#
+# HS Development Kit SoC.
+#
+# Contains a single ARC HS5 core (ARCv3).
+#
+
+source [find cpu/arc/hs5.tcl]
+
+set _coreid 0
+set _dbgbase [expr {$_coreid << 13}]
+
+# CHIPNAME will be used to choose core family (600, 700 or EM). As far as
+# OpenOCD is concerned EM and HS are identical.
+set _CHIPNAME arc-em
```

Comment:
That `arc-em` nonsense needs to be addressed later as it is clearly misleading. If that's some generic ARC code, why do we call it ARC EM?